packages/apps/immutable-records/src/screens/Home/Home.css.ts

```diff
@@ -2,11 +2,122 @@ import { sprinkles } from '@kadena/react-ui/theme';
 
 import { style } from '@vanilla-extract/css';
 
+// TODO: find a way to integrat mono font with react-ui
+// sprinkles({
+//   fontFamily: '$mono',
+// }),
+
+export const outerContainer = style([
+  sprinkles({
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+  }),
+]);
+
 export const container = style([
+  sprinkles({
+    display: 'flex',
+  }),
+  {
+    // make sure footer fits in the viewport
+    height: 'calc(100vh - 72px)',
+  },
+]);
+
+export const leftSideBar = style([
+  {
+    marginLeft: '1.5rem',
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    width: '200px',
+  },
+]);
+
+export const weekIndicatorBox = style([
+  {
+    background: '#372A46',
+    width: '6rem',
+    color: '#7D609F',
+    fontSize: '1.375rem',
+    fontWeight: 700,
+    textAlign: 'center',
+    padding: '0.25rem',
+  },
+]);
+
+export const weekIndicator = style([
+  {
+    marginTop: '1rem',
+    color: '#B1A0C5',
+    fontSize: '5rem',
+    lineHeight: '5rem',
+    fontWeight: 700,
+  },
+]);
+
+export const logoClassH1 = style([
+  {
+    color: '#FFF',
+    fontSize: '1.86669rem',
+    fontWeight: 700,
+    margin: '0 0 0.5rem 0',
+  },
+]);
+
+export const logoClassP = style([
+  {
+    color: '#B1A0C5',
+    fontSize: '1.25rem',
+    fontWeight: 400,
+    letterSpacing: '-0.025rem',
+    lineHeight: '1.5rem',
+    margin: '0 0 1.3rem 0',
+  },
+]);
+
+export const main = style([
   sprinkles({
     margin: '$4',
```

Comment:
hm..als je overal gewoon 'px' gebruikt. zou ik dat bij deze denk ik ook doen.
je zit zover van de react-ui af, dan zou ik het eigenlijk helemaal niet gebruiken.
anders veranderd opeens deze variabele en klopt er niets van je design.

ik zou wel aan vanilla extract vasthouden, want dat gebruiken we nu overal